app/filters.js

```diff
@@ -417,5 +417,10 @@ filters.falsify = (input) => {
     return last
   }
 
+  filters.push = (array, item) => {
+    array.push(item)
+    return array
+  }
+
```

Comment:
I think this mutates the array, which could be unpredictable. This is what I do on Register:

```
filters.push = (array, item) => {
  let newArray = [...array]
  newArray.push(_.cloneDeep(item)) // clone needed to stop this mutating original
  return newArray
}
```